### PR TITLE
Add Gitea v1.9

### PR DESCRIPTION
**Pull request by dschmidt**

### CODE REVIEW

File: v1.9/manifest.tmpl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+image: webhippie/gitea:latest
```
[the remark this answers is not in the document]
Comment (dschmidt) [reply v1.9/manifest.tmpl]: Missed that, good catch.

**Comment by dschmidt** [comment]
> Thanks for the pr, I guess you have not synced the available config options?

Cough. Nope, I read the complete backlog and didn't see any change mentioned. But you are right, I will check it now - was too late for that yesterday :-)

**Comment by tboerger** [comment]
Should I do the sync? Or do you volunteer for it? :)

**Comment by dschmidt** [comment]
Well, I started it and I should probably finish it ;-)

Hope I get to it this week

**Pull request merged**